src/payment/strategies/moneris/moneris-payment-strategy.ts

```diff
@@ -112,25 +114,36 @@ export default class MonerisPaymentStrategy implements PaymentStrategy {
         return Promise.resolve(this._store.getState());
     }
 
-    private _createIframe(containerId: string, profileId: string, testMode: boolean, style?: MonerisStylingProps): HTMLIFrameElement {
+    private _createIframe(containerId: string, initializationData: MonerisInitializationData, testMode: boolean, style?: MonerisStylingProps): HTMLIFrameElement {
         const container = document.getElementById(containerId);
 
         if (!container) {
             throw new InvalidArgumentError('Unable to create iframe without valid container ID.');
         }
 
         const iframe = document.createElement('iframe');
-        // Example CSS styling as provided by Moneris' documentation
-        const cssBody = style?.cssBody ?? 'background:white;';
-        const cssTextbox = style?.cssTextbox ?? 'border-width:2px;';
-        const csstextboxPan = style?.cssTextboxPan ?? 'width:140px;';
-        const cssTextboxExpiry = style?.cssTextboxExpiry ?? 'width:40px;';
-        const csstexboxCvd = style?.csstexboxCvd ?? 'width:40px';
+        const monerisQueryParams: Required<MoneriesHostedFieldsQueryParams> = {
```

Comment:
Since all the options are now mandatory in `MoneriesHostedFieldsQueryParams`, there's no need to wrap it in `Required`.

```suggestion
        const monerisQueryParams: MoneriesHostedFieldsQueryParams = {
```